src/lib/db/event-store.test.ts

```diff
@@ -29,39 +30,24 @@ test('Trying to get events by name if db fails should yield empty list', async (
     const events = await store.searchEvents({ type: 'application-created' });
     expect(events).toBeTruthy();
     expect(events.length).toBe(0);
+    await db.destroy();
 });
 
-test.each([
-    {
-        createdAt: formatRFC3339(subHours(new Date(), 1)),
-        expectedCount: 1,
-    },
-    {
-        createdAt: formatRFC3339(subHours(new Date(), 23)),
-        expectedCount: 1,
-    },
-    {
-        createdAt: formatRFC3339(subHours(new Date(), 25)),
-        expectedCount: 0,
-    },
-])(
-    'Find unnanounced events is capped to last 24hs',
-    async ({ createdAt, expectedCount }) => {
-        const db = await dbInit('events_test', getLogger);
-        const type = 'application-created' as const;
-        const insertQuery = db.rawDatabase('events');
-        await insertQuery
-            .insert({
-                type,
-                created_at: createdAt,
-                created_by: 'a test',
-                data: { name: 'test', createdAt },
-            })
-            .returning(['id']);
+test('Find unnanounced events is capped to 500', async () => {
```

Comment:
Does this test name match the content of the test now? Based on the expect I don't think so?

Reply:
Oh, no it doesn't :)

Reply:
```suggestion
// We might want to cap this to 500 and this test can help checking that
test('Find unannounced events returns all events', async () => { 
```